.travis.yml

```diff
@@ -1,8 +1,5 @@
 language: go
 
-services:
-  - mongodb
```

Comment:
It is, but my earlier integration added a second copy of it, so this deletes the extra copy. :)